plugins/com.google.cloud.tools.eclipse.appengine.standard.java8.test/src/com/google/cloud/tools/eclipse/appengine/standard/java8/AppEngineXsltTransformTest.java

```diff
@@ -153,21 +152,15 @@ private Document transform(String templateFile, String inputValue) {
       }
       InputStream documentStream =
           new ByteArrayInputStream(inputValue.getBytes(StandardCharsets.UTF_8));
-      transformed = Xslt.applyXslt(documentStream, xslPath.openStream());
-      DocumentBuilderFactory builderFactory = DocumentBuilderFactory.newInstance();
-      builderFactory.setNamespaceAware(true);
-      DocumentBuilder builder = builderFactory.newDocumentBuilder();
-      return builder.parse(transformed);
+      try (InputStream transformed = Xslt.applyXslt(documentStream, xslPath.openStream())) {
```

Comment:
This is an improvement over the previous.  I think the stream from `xslPath.openStream()` needs to be closed separately.